cpg_workflows/jobs/picard.py

```diff
@@ -18,6 +18,48 @@
 from cpg_workflows.utils import can_reuse, exists
 
 
+def blacklist_intervals(
+    b: hb.Batch,
+    job_attrs: dict | None = None,
+    source_intervals_path: Path | None = None,
+    blacklist_intervals_path: Path | None = None,
+) -> Job | None:
+    """
+    Add a job that subtracts the blacklisted intervals from the source intervals.
+
+    @param b: Hail Batch object,
+    @param job_attrs: attributes for Hail Batch job,
+    @param source_intervals_path: path to source intervals to use in joint calling.
+        Would check for config if not provided.
+    @param blacklist_intervals_path: path to blacklist intervals to exclude. 
+        Would check for config if not provided.
+
+    The job calls picard IntervalListTools to subtract the blacklist interval list
+    from the source interval list.
+    """
+    sequencing_type = get_config()['workflow']['sequencing_type']
+    source_intervals_path = source_intervals_path or reference_path(f'broad/{sequencing_type}_calling_interval_lists')
+    blacklist_intervals_path = blacklist_intervals_path or reference_path('broad/blacklist_intervals') #TODO add this to config (default?)
```

Comment:
the standard one here would be `reference_path('hg38_telomeres_and_centromeres')`?

Reply:
once it gets successfully copied...